python-port/setup.py

```diff
@@ -0,0 +1,39 @@
+import setuptools
+
+
+def readme():
+    with open("README.rst") as fh:
+        return fh.read()
+
+setuptools.setup(name="sendpy",
+      version="1.0",
+      description="Email and text notification program",
+      long_description=readme(),
+      long_description_content_type="text/markdown",
+      url="https://github.com/tdulcet/Send-Msg-CLI",
+      author="Daniel Connelly and Teal Dulcet",
+      author_email="connellyd2050@gmail.com",
+      license="GPL",
+      packages=["sendpy"],
+      scripts=["bin/sendpy"],
+      classifiers=[
+          "Programming Language :: Python :: 3.6",
+          "Programming Language :: Python :: 3.7",
+          "Programming Language :: Python :: 3.8",
+          "Programming Language :: Python :: 3.9",
+          "Programming Language :: Python :: 3.10",
+          "Programming Language :: Python :: 3.11",
+          "Programming Language :: Python :: 3.12",
+          "License :: OSI Approved :: GNU General Public License (GPL)",
+          "Operating System :: OS Independent",
+      ],
+      keywords="sendpy email e-mail send text notification g-mail gmail",
+      python_requires=">=3.6",
+      include_package_data=True,
+      zip_safe=False,
```

Comment:
I think this is no longer needed ([source](https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html))

Reply:
Thanks, I removed this line.